src/api/app/assets/javascripts/webui/datatables.js

```diff
@@ -44,3 +44,26 @@ function initializeRemoteDatatable(cssSelector, params) {
 
   $(cssSelector).dataTable(newParams);
 }
+
+function labelFiltering() {
+  var table = $('#packages-table').DataTable();
+  var labelColumn = table.column('labels:name');
+  var clear = $('#label-clear');
+  $('.obs-dataTable').parent().on('click', '.label-filter', function(e) {
+    e.preventDefault();
+    e.stopPropagation();
+    var label = e.target.parentElement.dataset.label;
+    clear.html('');
+    clear.data('label', label);
+
+    if (label === labelColumn.search())
+      labelColumn.search('').draw();
+    else
+      labelColumn.search(label).draw();
+
+    if (labelColumn.search() !== '')
+      clear.html('Clear label filter');
```

Comment:
My impressions:
- The "_Clear label filter_" goes a bit unnoticed
- If two packages have more than one label in common. Which one did I click?

I would make more obvious that there is a label selected and which label it is.

Scenario (see screenshot): I go back to a browser tab I opened a few hours ago. The page says there are 47 packages but I see only 2. Why? It's not visually clear.

<img width="1481" height="771" alt="Screenshot 2025-11-12 at 16-03-45 Show home Admin - Open Build Service" src="https://github.com/user-attachments/assets/e400f369-557b-47f6-a59b-c5c01c3c9cfe" />